threshsign/include/threshsign/eddsa/EdDSAMultisigSigner.h

```diff
@@ -15,6 +15,8 @@
 #include "EdDSAThreshsignKeys.h"
 #include "crypto/eddsa/EdDSASigner.hpp"
 
+using concord::crypto::openssl::eddsa::EdDSASigner;
```

Comment:
dont use using in a header

Reply:
Removed in https://github.com/arc-vmware/concord-bft/pull/6/commits/87f4edd0f4f5d72b422cef50f4a8eeeee8b28bc2